ios/Framework/Core/Auth/LoginScreenlet/LoginScreenlet.swift

```diff
@@ -72,7 +72,7 @@ public class LoginScreenlet: BaseScreenlet, BasicAuthBasedType {
 			viewModel.userName = SessionContext.currentContext?.basicAuthUsername
 			viewModel.password = SessionContext.currentContext?.basicAuthPassword
 
-			let userAttributes = SessionContext.currentContext!.userAttributes
```

Comment:
Add a computed property called `userAttributes` to maintain backwards compatibility